public/app/core/utils/fetch.ts

```diff
@@ -57,9 +57,18 @@ const putHeaderParser: HeaderParser = parseHeaderByMethodFactory('put');
 const patchHeaderParser: HeaderParser = parseHeaderByMethodFactory('patch');
 
 const headerParsers = [postHeaderParser, putHeaderParser, patchHeaderParser, defaultHeaderParser];
+const nonSafeCharacters = /[^\u0000-\u00ff]/g;
+
+function safeValue(v: string) {
+  return nonSafeCharacters.test(v) ? encodeURI(v) : v;
```

Comment:
What if we just always `safeValue()`? I would also consider using `sanitize()` in the name.